rental_info_tab.py

```
@@ -0,0 +1,739 @@
+import sys
```

Ruff (0.11.9)

`sys` imported but unused

Remove unused import: `sys`

(F401)